core/chains/evm/gas/fixed_price_estimator.go

```diff
@@ -50,20 +56,32 @@ func (f *fixedPriceEstimator) Start(context.Context) error {
 	}
 	return nil
 }
-func (f *fixedPriceEstimator) Name() string                                          { return f.lggr.Name() }
-func (f *fixedPriceEstimator) Ready() error                                          { return nil }
-func (f *fixedPriceEstimator) HealthReport() map[string]error                        { return map[string]error{} }
-func (f *fixedPriceEstimator) Close() error                                          { return nil }
-func (f *fixedPriceEstimator) OnNewLongestChain(_ context.Context, _ *evmtypes.Head) {}
 
-func (f *fixedPriceEstimator) GetLegacyGas(_ context.Context, _ []byte, gasLimit uint32, maxGasPriceWei *assets.Wei, _ ...feetypes.Opt) (gasPrice *assets.Wei, chainSpecificGasLimit uint32, err error) {
-	gasPrice = f.config.PriceDefault()
-	gasPrice, chainSpecificGasLimit = capGasPrice(gasPrice, maxGasPriceWei, f.config.PriceMax(), gasLimit, f.config.LimitMultiplier())
-	return
+func (f *fixedPriceEstimator) GetLegacyGas(_ context.Context, _ []byte, gasLimit uint32, maxGasPriceWei *assets.Wei, _ ...feetypes.Opt) (*assets.Wei, uint32, error) {
+	gasPrice, chainSpecificGasLimit, err := commonfee.CalculateFee(gasLimit, maxGasPriceWei.ToInt(), f.config.PriceDefault().ToInt(), f.config.PriceMax().ToInt(), f.config.LimitMultiplier())
+	return assets.NewWei(gasPrice), chainSpecificGasLimit, err
 }
 
-func (f *fixedPriceEstimator) BumpLegacyGas(_ context.Context, originalGasPrice *assets.Wei, originalGasLimit uint32, maxGasPriceWei *assets.Wei, _ []EvmPriorAttempt) (gasPrice *assets.Wei, gasLimit uint32, err error) {
-	return BumpLegacyGasPriceOnly(f.config, f.lggr, f.config.PriceDefault(), originalGasPrice, originalGasLimit, maxGasPriceWei)
+func (f *fixedPriceEstimator) BumpLegacyGas(
+	_ context.Context,
+	originalGasPrice *assets.Wei,
+	originalGasLimit uint32,
+	maxGasPriceWei *assets.Wei,
+	_ []EvmPriorAttempt,
```

Comment:
Can we drop any of these unused args?
```suggestion
```

Reply:
I am looking into how we can drop this unused args. Right now they are used to satisfy interface, but not sure if we can drop them 🤔